### ISSUE TITLE
Homebrew: Both recipes simultaneously

### ISSUE THREAD

**Issue by ylluminate** [utmapp/UTM#5739]
I find myself frequently interleaving between UTM and UTM-beta on `Homebrew` due to wanting to test and then also be able to run the stable. Could we please have support to run both versions simultaneously via a naming mechanism that would facilitate both an `/Applications/UTM.app` and an `/Applications/UTM-Beta.app` or some such naming architecture that can facilitate both?

**Comment by kmatheussen** [comment]
> Does anyone else think this makes sense?

Yes. Would this also mean that you can't share VMs between UTM and UTM-Beta (unless you copy a VM from one to another)? In case, that would be a nice way to test without risking to destroy existing VMs.
